lessons/04-structured-output/translations/README.fr.md

```diff
@@ -0,0 +1,561 @@
+# Leçon 4 : Sorties structurées
+
+Dans ce chapitre, vous apprendrez :
+
+* Ce que sont les *sorties structurées* et comment les utiliser efficacement.
+* Comment extraire des informations à partir des instructions (prompts) et les intégrer dans la sortie.
+* Générer différents types de formats de sortie, tels que JSON, pour faciliter leur utilisation par des services.
+
+## Configuration
+
+Si ce n'est pas encore fait, configurez votre environnement de développement. Voici comment procéder : [Configurez votre environnement](/docs/setup/README.md).
+
+## Ressources associées
+
+Cela vaut la peine de revoir la vidéo sur l'ingénierie des prompts, car elle pose les bases de ce que vous allez apprendre dans ce chapitre.
+
+[![Watch a short video about prompt engineering](https://img.youtube.com/vi/gQ6TlyxBmWs/0.jpg)](https://www.youtube.com/watch?v=gQ6TlyxBmWs\&list=PLlrxD0HtieHi5ZpsHULPLxm839IrhmeDk\&index=3)
+
+*Cette vidéo propose une introduction à l'amélioration de vos compétences en matière de "prompt", en vous apprenant à donner des instructions plus claires et plus efficaces à l'IA pour obtenir de meilleurs résultats.*
+
+*🎥 Cliquez sur l'image ci-dessus pour regarder une courte vidéo sur l'ingénierie des prompts*
+
+💼 Présentation : [Ingénierie des prompts](/videos/slides/02-prompt-engineering.pptx)
+
+## Récit - De Charybde en Scylla
+
+> > [!NOTE]
```

Comment:
Double `>` here, not sure why? maybe trying to recompile the MD node after translation to see the same node structure was kept might help with issues like these?